src/NUnitFramework/framework/Attributes/RandomAttribute.cs

```diff
@@ -42,6 +43,11 @@ public class RandomAttribute : DataAttribute, IParameterDataSource
         private RandomDataSource _source;
         private int _count;
 
+        /// <summary>
+        /// If true, no value will be repeated
```

Comment:
Nit: the general style is to end both sentences (this is one) and fragments with a period. I don't know if this is in the coding standard but NUnit mostly follows it and the .NET BCL always follows it.